app/views/works/show.html.erb

```diff
@@ -1,4 +1,5 @@
 <h1 class="h3 mb-4"><%= @work_form.title %></h1>
-<%= render Works::Show::TableComponent.new(label: 'Title', id: 'title-table') do |component| %>
-    <% component.with_row(label: 'Title', value: @work_form.title) %>
+<%= render Works::Show::TableComponent.new(label: 'Description', id: 'description-table') do |component| %>
```

Comment:
Title and related links go in separate tables.

Reply:
OK, will change.